Patch fixes (#120)
DiamondLightSource/python-copier-template: .github/CONTRIBUTING.md

## Checking your changes before making a PR

The template has tests for:

- Generating a new project from scratch
- Updating the [example project](https://github.com/DiamondLightSource/python-copier-template-example)
- Checking that both of the above produce the same results

> However, this does not test whether processes like CI and docs work correctly. To ensure that this can be checked, you can:

Reviewer: ```suggestion
However, this does not test whether processes like CI and docs work correctly. You can ensure that is checked by:
```

It is recommended that developers use a [vscode devcontainer](https://code.visualstudio.com/docs/devcontainers/containers). This repository contains configuration to set up a containerized development environment that suits its own needs.

For more information on common tasks like setting up a developer environment, running the tests, and setting a pre-commit hook, see the [How-to guides](https://diamondlightsource.github.io/python-copier-template/main/how-to.html)

## Making a Tagged GitHub Release

> When making changes to the `catalog-info.yaml` file, a release **must** be made. The Backstage Developer Portal produces its UI for the template using this file from the `main` branch whereas copier takes the template from the latest tag. Therefore, any differences between the copier variables could cause a breaking change so a major version release should be made.

Reviewer: ```suggestion
When making changes to the `catalog-info.yaml` file, a release **must** be made. The Backstage Developer Portal produces its UI for the template using this file from the `main` branch whereas copier takes the template from the latest tag. Therefore, any differences between the copier variables could cause a breaking change so a release should be made as soon as possible.
```
